webapp/app/protected/users/user/template.hbs

```diff
@@ -1,77 +1,79 @@
 <div class="section">
-	<h4>
-		{{#if model.isAdmin }}
-			{{#link-to 'protected.users'}} Users {{/link-to}} -
-		{{/if}}
-		{{#edit-text
-			class='in-bl'
-			type="text"
-			confirmation=false
-			textInput=model.firstName
-			textInputPlaceholder="First name"
-			onClose="changeFirstName"
-		}}
-			{{model.firstName}}
-		{{/edit-text}}
-		{{#edit-text
-			class='in-bl'
-			type="text"
-			confirmation=false
-			textInput=model.lastName
-			textInputPlaceholder="Last name"
-			onClose="changeLastName"
-		}}
-			{{model.lastName}}
-		{{/edit-text}}
+  <h4>
+    {{#if model.isAdmin }}
```

Comment:
You probably want to check the connected user (`session.user.isAdmin`). Not the displayed user.